### PR TITLE
T16682

### PR DESCRIPTION
**Pull request by smspillaz**
https://phabricator.endlessm.com/T16682

### CODE REVIEW

File: lib/flatpak-installation.c [hunk 1]
```diff
@@ -2009,3 +2009,59 @@ flatpak_installation_list_installed_related_refs_sync (FlatpakInstallation *self
...
+ * objects which were attached to that ref will be removed. This is useful if
+ * you pulled a flatpak refs using flatpak_installation_install_full() and
+ * specified %FLATPAK_INSTALL_FLAGS_NO_DEPLOY but then decided not to
+ * deploy the refs later on and want to remove the local refs to prevent them
+ * from taking up disk space.
+ *
+ * Returns: %TRUE on success
+ */
+gboolean
+flatpak_installation_cleanup_local_refs_sync (FlatpakInstallation *self,
```
Comment (pwithnall) [inline lib/flatpak-installation.c]: There will need to be a corresponding change to `libflatpak0.symbols` on the `debian-master` branch for this new API (and for the other new API introduced in this branch).

**Comment by pwithnall** [comment]
These changes have all since appeared in the flatpak 0.10.0 release, so it might make more sense to just rebase to flatpak 0.10.0 and get a bunch of other fixes too. I suggest we do that for 3.4, and don’t backport it to 3.3 (since these changes are needed for [T16682](https://phabricator.endlessm.com/T16682), which is not landing on 3.3).

**Comment by nedrichards** [comment]
fwiw I would *love* to get a rebase to 0.10 as the flatpak/flathub community are using and expecting features available in newer releases (e.g. host icon support)

**Comment by pwithnall** [comment]
> fwiw I would love to get a rebase to 0.10 as the flatpak/flathub community are using and expecting features available in newer releases (e.g. host icon support)

⇒ https://phabricator.endlessm.com/T20056

**Comment by smspillaz** [comment]
Closing, as we decided to rebase flatpak instead

**Pull request closed without merging**